GVRf/Framework/src/org/gearvrf/GVRMesh.java

```diff
@@ -291,31 +283,32 @@ public GVRMesh getBoundingBox() {
         return new GVRMesh(getGVRContext(), NativeMesh.getBoundingBox(getPtr()));
     }
 
-    private static void isNonNull(Object data) {
-        if (data == null) {
-            throw new IllegalArgumentException(
-                    "The input array should not be null.");
-        }
+    private void checkValidFloatVector(String parameterName, String key,
```

Comment:
Sorry - you are testing two different parameters here: you should have two different parameter names.

    void checkValidFloatVector(String keyName, String key, String vectorName, float[] vector, int expectedComponents)